BGL/include/CGAL/boost/graph/properties_OpenMesh.h

```diff
@@ -231,8 +233,8 @@ class OM_point_pmap
 #if defined(CGAL_USE_OM_POINTS)
     const_cast<OpenMesh&>(*pm.sm_).set_point(v,p);
 #else
-    const_cast<OpenMesh&>(*pm.sm_).set_point
-      (v, typename OpenMesh::Point((float)p[0], (float)p[1], (float)p[2]));
+    typedef typename CGAL::Kernel_traits<typename OpenMesh::Point>::type FT;
+    const_cast<OpenMesh&>(*pm.sm_).set_point(v, typename OpenMesh::Point(FT(p[0]), FT(p[1]), FT(p[2])));
```

Comment:
@sloriot @afabri A bit confused by the `float`: in the properties_OM, we use `EPICK::Point_3` when not using `OpenMesh::Point`. Was it to have symmetry with results obtained with OpenMesh for when only `float`-based `OpenMesh::Point`s were used?

Reply:
Seems so:
https://www.graphics.rwth-aachen.de/media/openmesh_static/Documentations/OpenMesh-6.0-Documentation/a00131.html#a57c3a0075b2f50719679de132aa772b1

Reply:
It still is, but there also exists `DefaultTraitsDouble`, where `Point` is `vec3d`, which we do use in just a few places. In this case, we cast to float even though our Point is EPICK::Point_3 and the OpenMesh's scalar type is double.